src/components/picker/helpers/usePickerSelection.tsx

```diff
@@ -29,14 +29,8 @@ const usePickerSelection = (props: UsePickerSelectionProps) => {
   [onChange]);
 
   const toggleItemSelection = useCallback((item: PickerSingleValue) => {
-    let newValue;
     const itemAsArray = [item];
-    if (!migrate) {
```

Comment:
You can't remove code from public without changing the major version - this should be part of the `v8` effort